package.json

```diff
@@ -22,9 +22,9 @@
     "start": "npm config set quiet=true; eleventy --serve",
     "lerna": "lerna",
     "test": "node scripts/test.js",
-    "test:watch": "npm config set group=default && npm config set element=\"*\" && web-test-runner \"elements/*/test/$npm_config_element.spec.js\" --node-resolve --playwright --watch --group=$npm_config_group",
+    "test:watch": "npm config set group=default && npm config set element=\"*\" && web-test-runner \"elements/*/test/$npm_config_element.spec.js\" --node-resolve --watch --group=$npm_config_group",
     "test:build:watch": "npm config set element=\"*\" && npm run build \"$npm_config_element\" && npm run test:watch",
-    "test:ci": "web-test-runner --node-resolve --playwright",
+    "test:ci": "web-test-runner --node-resolve",
```

Comment:
Because we are specifying the playwright instance in the 'browser' config object below, we don't need to specify it in our web-test-runner command.  web-test-runner will throw an error if we try to set the `--playwright` flag in addition to using the `playwrightLauncher` plugin below.

Reply:
Naw, that's cool if we moved it to config. 👍